### ISSUE TITLE
Update website

### ISSUE THREAD

**Issue by futagoza** [pegjs/pegjs#578]
I've just finished merging the website into the main repo as part of a set of changes that will:

- add the test suites to the website so I can do automated browser testing
- read the markdown docs dynamically from https//pegjs.org/docs/#/

These are issues that were open on [@pegjs/website](https://github.com/pegjs/website) before I archived it:

- pegjs/website#20
- pegjs/website#23
- pegjs/website#27

**Comment by futagoza** [comment]
Decided to:

- Use [VuePress](https://vuepress.vuejs.org/) as the static site generator
- Move to Now v2
- Not focus on IE support for the online editor's "Download parser" button

**Comment by polkovnikov-ph** [comment]
Yay, what a win!

**Comment by futagoza** [comment]
win?

**Comment by polkovnikov-ph** [comment]
Yes, a win. As if you played a game for years and won a grand prize.

**Comment by mikeaustin** [comment]
I was curios about how to run the current web site locally–I wanted to play around with how jsDump outputs objects. If I go into website and do npm install, npm start, I get:

Error: Cannot find module '@pegjs/bundle-generator'

**Comment by futagoza** [comment]
The website folder is there for _easier to access reference_ while I rebuild the site in VuePress, and hasn't been updated for a while.

**Comment by StoneCypher** [comment]
The marked complete URL doesn't exist, and this is about a two hour job with github pages.  There's no difficulty in supporting IE; I don't know why you keep saying that.  I'm really worried that there's a bunch of portability timebombs in `0.11.0`

I'm not really sure, but it looks like the `zeit` account attached might be gone

Let's stop using exotic tools and switch to standard stuff that will stay up please

**Comment by StoneCypher** [comment]
Also, [I found what appears to be an unrelated copy of the same build](https://pegjs-6tlfvclbp.now.sh/online), and it appears to be identical to the one in production

This suggests that any IE support problems are about damage to the engine, and are unrelated to the site
